app/views/courses/_status_panel.html.erb

```diff
@@ -29,8 +29,11 @@
     <% end %>
   </p>
 
-  <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m">
-  <p class="govuk-body">Last published:<br><%= formatted_date(@course.last_published_at) %></p>
+  <% if @course.last_published_at.present? %>
+    <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m">
+    <p class="govuk-body">Last published:<br><%= @course.last_published_at %></p>
```

Comment:
I don't particularly like here that someone who isn't familiar with Draper or doesn't know that `@course` is the decorated course can find themselves scratching their head as to how exactly `last_published_at` is being shown to the user as `5 March 2019` instead of `2019-03-05T14:42:34Z`.

I see that Draper doesn't require that you implement a decorated method that is called exactly like the actual model attribute.

What do you think of `formatted_last_published_at`?

This is a minor suggestion.

Reply:
I agree and had that to begin with but didn’t like the length of it ha. I’ll change it

Reply:
I think it's long too, it's just the first thing that came to mind...

Reply:
I'm not bothered by long names, and personally think that this one is Good Enough (:tm:). Also, good suggestion to make it clearer which attributes are specially formatted.